client/ayon_usd/version.py

```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 """Package declaring AYON addon 'ayon_usd' version."""
 name = "ayon_usd"
-__version__ = "1.0.4-dev.37"
+__version__ = "1.0.4-dev.38"
```

Comment:
This should match the version in the `package.py`
```suggestion
__version__ = "0.1.0-alpha"
```